src/main.c

```diff
@@ -357,6 +365,12 @@ static void focal_create_main_window(GApplication* app, FocalApp* fm)
 	gtk_widget_show_all(fm->mainWindow);
 
 	app_header_set_event(FOCAL_APP_HEADER(fm->header), NULL);
+
+	fm->sync_timer_id = g_timeout_add_seconds(
+		120 /* sync interval, seconds */,
+		G_SOURCE_FUNC(sync_func),
+		fm);
```

Comment:
Comment is a little redundant thanks to the `_seconds` in the API. Then it can easily fit on one line and save vertical space. Not critical, just since we're discussing this...

Reply:
Agreed, thanks for your review O:).